eternalcore-core/src/main/java/com/eternalcode/core/feature/randomteleport/RandomTeleportService.java

```diff
@@ -74,10 +78,26 @@ private CompletableFuture<Location> getSafeRandomLocation(World world, int attem
             return CompletableFuture.failedFuture(new RuntimeException("Cannot find safe location"));
         }
 
-        int radius = this.randomTeleportSettings.randomTeleportRadius();
+        int radius = 0;
+        if (this.randomTeleportSettings.randomTeleportType() == RandomTeleportType.STATIC_RADIUS) {
+            radius = this.randomTeleportSettings.randomTeleportRadius();
+        }
+
+        if (this.randomTeleportSettings.randomTeleportType() == RandomTeleportType.WORLD_BORDER_RADIUS) {
+            WorldBorder worldBorder = world.getWorldBorder();
```

Comment:
worldborder może być nullem to na pewno, przynajmniej tak było na 1.8

Reply:
Nie, worldborder zawsze będzie ustawiony, minecraft po odpaleniu silnika ustawia border na defaultową wartość. 


```java
    /**
     * Gets the world border for this world.
     *
     * @return The world border for this world.
     */
    @NotNull
    public WorldBorder getWorldBorder();
```